src/Output/ShellOutput.php

```diff
@@ -176,11 +176,16 @@ private function initFormatters()
     {
         $formatter = $this->getFormatter();
         $errorFormatter = $this->getErrorOutput()->getFormatter();
+        $grayExists = $this->grayExists();
 
+        $formatter->setStyle('info', new OutputFormatterStyle('white', 'blue', ['bold']));
+        $errorFormatter->setStyle('info', new OutputFormatterStyle('white', 'blue', ['bold']));
         $formatter->setStyle('warning', new OutputFormatterStyle('black', 'yellow'));
         $errorFormatter->setStyle('warning', new OutputFormatterStyle('black', 'yellow'));
         $formatter->setStyle('error', new OutputFormatterStyle('white', 'red', ['bold']));
         $errorFormatter->setStyle('error', new OutputFormatterStyle('white', 'red', ['bold']));
+        $formatter->setStyle('whisper', new OutputFormatterStyle($grayExists ? 'gray' : 'blue'));
+        $errorFormatter->setStyle('whisper', new OutputFormatterStyle($grayExists ? 'gray' : 'blue'));
```

Comment:
PoC:
```suggestion
        $formatter->setStyle('info', $outputFormatter = new OutputFormatterStyle('white', 'blue', ['bold']));
        $errorFormatter->setStyle('info', $outputFormatter);
        $formatter->setStyle('warning', $outputFormatter = new OutputFormatterStyle('black', 'yellow'));
        $errorFormatter->setStyle('warning', $outputFormatter);
        $formatter->setStyle('error', $outputFormatter = new OutputFormatterStyle('white', 'red', ['bold']));
        $errorFormatter->setStyle('error', $outputFormatter);
        $formatter->setStyle('whisper', $outputFormatter = new OutputFormatterStyle($grayExists ? 'gray' : 'blue'));
        $errorFormatter->setStyle('whisper', $outputFormatter);
```

I'm not happy with consecutive `$outputFormatter` usage, but the idea here is to reuse style between `$formatter` and `$errorFormatter`. Right now it's possible to modify style in one formatter and leave it untouched in the other one, which may lead to inconsistency. It could be improved IMHO.